src/catalog/catalog.cpp

```diff
@@ -523,30 +516,28 @@ std::unique_ptr<storage::DataTable> Catalog::CreateMetricsCatalog(
 std::unique_ptr<catalog::Schema> Catalog::InitializeTablesSchema() {
   const std::string not_null_constraint_name = "not_null";
 
-  auto id_column = catalog::Column(
-      type::Type::INTEGER, type::Type::GetTypeSize(type::Type::INTEGER),
-      "table_id", true);
+  auto id_column = catalog::Column(type::Type::INTEGER,
+      type::Type::GetTypeSize(type::Type::INTEGER), "table_id", true);
   id_column.AddConstraint(
       catalog::Constraint(CONSTRAINT_TYPE_NOTNULL, not_null_constraint_name));
 
-  auto name_column =
-      catalog::Column(type::Type::VARCHAR, max_name_size, "table_name", true);
+  auto name_column = catalog::Column(type::Type::VARCHAR, max_name_size,
+      "table_name", true);
   name_column.AddConstraint(
       catalog::Constraint(CONSTRAINT_TYPE_NOTNULL, not_null_constraint_name));
 
-  auto database_id_column = catalog::Column(
-      type::Type::INTEGER, type::Type::GetTypeSize(type::Type::INTEGER),
-      "database_id", true);
+  auto database_id_column = catalog::Column(type::Type::INTEGER,
+      type::Type::GetTypeSize(type::Type::INTEGER), "database_id", true);
   database_id_column.AddConstraint(
       catalog::Constraint(CONSTRAINT_TYPE_NOTNULL, not_null_constraint_name));
 
-  auto database_name_column = catalog::Column(
-      type::Type::VARCHAR, max_name_size, "database_name", true);
+  auto database_name_column = catalog::Column(type::Type::VARCHAR,
```

Comment:
I actually didn't touch this part, these changes are due to formatting changes only